frontend/packages/data-portal/app/components/Run/AnnotationObjectTable/components/InfoLink/InfoLink.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'app/components/Link'
+import {
+  GO,
+  GO_PATTERN,
+  UNIPROTKB,
+  UNIPROTKB_PATTERN,
+} from 'app/constants/annotationInfoLinks'
+
+export function InfoLink({ id }: { id: string }) {
```

Comment:
nit - should this be called ObjectIdLink or something, since it's only for displaying object IDs?

Reply:
Changed, thanks!